charts/seed-bootstrap/templates/gardener-seed-admission-controller/mutatingwebhookconfiguration.yaml

```
@@ -0,0 +1,37 @@
+apiVersion: admissionregistration.k8s.io/v1
```

```suggestion
apiVersion: admissionregistration.k8s.io/v1beta1
```

Can you please use the `v1beta1` version to also serve seed clusters with a lower Kubernetes version? I'm seeing

```
time="2020-05-22T07:24:42+02:00" level=error msg="Seed bootstrapping failed: failed to apply manifests: 1 error occurred: could not apply object of kind \"MutatingWebhookConfiguration\" \"default/gardener-seed-admission-controller\": no matches for kind \"MutatingWebhookConfiguration\" in version \"admissionregistration.k8s.io/v1\"" seed=aws
```